### PR TITLE
Remove push to CAPI app collection

### PR DESCRIPTION
**Pull request by tuladhar**
Node exporter is part of default apps. Thus, it's conflicting to also push to CAPI app collection which already uses default apps.

Towards roadmap: https://github.com/giantswarm/roadmap/issues/1811

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -51,88 +51,3 @@ workflows:
...
-          app_collection_repo: "aws-app-collection"
-          requires:
-            - push-node-exporter-to-control-plane-app-catalog
-          filters:
-            branches:
-              ignore: /.*/
-            tags:
-              only: /^v.*/
-
-      - architect/push-to-app-collection:
```
Comment (QuentinBisson) [inline .circleci/config.yml:83]: Only CAPI collections should be removed

Comment (tuladhar) [reply .circleci/config.yml:83]: But, default apps is already deploying node-exporter for other providers which will conflict it's install again with app collection. Isn't that so?
- https://github.com/giantswarm/default-apps-vsphere/blob/main/helm/default-apps-vsphere/values.yaml
- https://github.com/giantswarm/default-apps-cloud-director/blob/main/helm/default-apps-cloud-director/values.yaml
- https://github.com/giantswarm/default-apps-aws/blob/master/helm/default-apps-aws/values.yaml

Exception is for azure where it's not part of default apps.
- https://github.com/giantswarm/default-apps-azure/blob/main/helm/default-apps-azure/values.yaml

Comment (QuentinBisson) [reply .circleci/config.yml:83]: default apps aws is for capa, not aws (i.e. vintage)

File: .circleci/config.yml [hunk 2]
```diff
@@ -51,88 +51,3 @@ workflows:
...
-          app_collection_repo: "shared-app-collection"
-          requires:
-            - push-node-exporter-to-control-plane-app-catalog
-          filters:
-            branches:
-              ignore: /.*/
-            tags:
-              only: /^v.*/
-
-      - architect/push-to-app-collection:
```
Comment (QuentinBisson) [inline .circleci/config.yml:69]: This is for vintage so it needs to stay

File: .circleci/config.yml (hunk 1 above)
Comment (tuladhar) [reply .circleci/config.yml:83]: Ah okay, got it. Thanks much! ❤️

**Comment by QuentinBisson** [comment]
Can you also remove vsphere?

**Review by QuentinBisson — APPROVED** [review]

**Pull request merged**